geode-core/src/integrationTest/java/org/apache/geode/distributed/LauncherMemberMXBeanIntegrationTest.java

```diff
@@ -122,29 +123,39 @@ public void showOSMetrics_reconstructsOSMetricsFromCompositeDataType()
     OSMetrics osMetrics = mbean.showOSMetrics();
     assertThat(osMetrics).isNotNull();
 
-    Long osMetricsCommittedMemory = osMetrics.getCommittedVirtualMemorySize();
-    float virtualMemoryRatio = osMetricsCommittedMemory.floatValue()
-        / ((Long) cds.get("committedVirtualMemorySize")).floatValue();
-
-    // On windows in particular, the memory value returned from the live bean has often already
-    // changed from the statically recorded value.
-    assertThat(virtualMemoryRatio).isCloseTo(virtualMemoryRatio, within(0.01F));
-
     // Verify conversion from CompositeData to OSMetrics
     assertThat(osMetrics.getArch()).isEqualTo(cds.get("arch"));
     assertThat(osMetrics.getAvailableProcessors()).isEqualTo(cds.get("availableProcessors"));
-    assertThat(osMetrics.getFreePhysicalMemorySize()).isEqualTo(cds.get("freePhysicalMemorySize"));
-    assertThat(osMetrics.getFreeSwapSpaceSize()).isEqualTo(cds.get("freeSwapSpaceSize"));
     assertThat(osMetrics.getMaxFileDescriptorCount()).isEqualTo(cds.get("maxFileDescriptorCount"));
     assertThat(osMetrics.getName()).isEqualTo(cds.get("name"));
-    assertThat(osMetrics.getOpenFileDescriptorCount())
-        .isEqualTo(cds.get("openFileDescriptorCount"));
-    assertThat(osMetrics.getProcessCpuTime()).isEqualTo(cds.get("processCpuTime"));
-    assertThat(osMetrics.getSystemLoadAverage()).isEqualTo(cds.get("systemLoadAverage"));
     assertThat(osMetrics.getTotalPhysicalMemorySize())
         .isEqualTo(cds.get("totalPhysicalMemorySize"));
     assertThat(osMetrics.getTotalSwapSpaceSize()).isEqualTo(cds.get("totalSwapSpaceSize"));
     assertThat(osMetrics.getVersion()).isEqualTo(cds.get("version"));
+
+    assertThat(tryConvergeVolatileOSMetrics("committedVirtualMemorySize",
```

Comment:
These assertions might be good candidates to use .as("") to describe the subject of the assertion.

Reply:
I thought of that too but couldn't come up with something that was not overly wordy and would have cluttered the code.

Reply:
An example: For line 133 consider something like:
```
.as("total swap space size")
```
It does make the code wordier, but it draws attention to the right code very quickly.